app/src/controllers/log/actionCreators.js

```diff
@@ -50,3 +51,8 @@ export const setPageLoadingAction = (isLoading) => ({
   type: SET_LOG_PAGE_LOADING,
   payload: isLoading,
 });
+
+export const updateHistoryEntryLocally = (testCaseHash, id, data) => ({
```

Comment:
The name of action doesn't clear represents its purpose. May be you need just action for setting issue to history entity.

Reply:
Fixed